lib/snapshot/BUILD.gn

```diff
@@ -423,8 +427,41 @@ action("compile_platform") {
          rebase_path(outputs, root_build_dir)
 }
 
+action("compile_platform") {
+  script = "//third_party/dart/tools/compile_platform.py"
+
+  visibility = [
+    ":kernel_platform_files"
+  ]
+  
+  sources = [
+    "$root_out_dir/flutter_patched_sdk/lib/libraries.json",
+  ]
+
+  outputs = [
+    "$root_out_dir/flutter_patched_sdk/platform_strong.dill",
```

Comment:
You probably want to use this file in frontend server when running in strong mode (server.dart:173)

Reply:
Fixed code to link with the correct platform dill file during aot builds.